vocabularies/voc4cat/0007122.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007122>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007122"^^xsd:token ;
+    skos:altLabel "offset correction; zero-point adjustment; baseline offset"@en ;
```

Comment:
In contrast to the other alt.labels "zero-point adjustment" is a process so does not fit here.